app/i18n/en/gen.php

```diff
@@ -14,7 +14,7 @@
 		'import' => 'Import',
 		'manage' => 'Manage',
 		'mark_read' => 'Mark as read',
-		'mark_favorite' => 'Mark as favourite',
+		'mark_favorite' => 'Mark as favorite',
```

Comment:
If the rest of the file is en_GB, then it is *favourite* https://en.wiktionary.org/wiki/favourite
*favorite* is en_US